db/migrate/20190303182743_create_taggings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTaggings < ActiveRecord::Migration[5.1]
+  def change
+    create_table :taggings do |t|
+      t.belongs_to :project, foreign_key: true
+      t.belongs_to :tag, foreign_key: true
+
```

Comment:
Is it possible to add indexes here and also in the tags table? like [so](https://stackoverflow.com/questions/223804/indexes-for-has-many-through). This'll help with the queries.

Reply:
@ankitkataria  this was just a primitive model we can add indexes.